docs/core/docker/visual-studio-tools-for-docker.md

```diff
@@ -0,0 +1,363 @@
+ms.date: 09/16/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 1f3b9a68-4dea-4b60-8cb3-f46164eedbbf
+---
+
+# Visual Studio Tools for Docker 
+Developing and debugging your application in a Docker container can be a ceremony of tasks to get setup with various tools. [Visual Studio Tools for Docker](https://visualstudiogallery.msdn.microsoft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4) helps you get past the hurdles and find the bugs using F5 to debug your application directly in a locally hosted Docker Container. 
+
+> [!NOTE]
+>The current version targets Linux Docker containers, with Windows Containers coming soon.
+
+## Prerequisites
+- [Microsoft Visual Studio 2015 Update 3](https://www.visualstudio.com/downloads/download-visual-studio-vs)
+- [Microsoft .NET Core 1.0.1 RTM VS 2015 Tooling Preview 2, with the 1.0.1 updates](https://go.microsoft.com/fwlink/?LinkID=827546)
+- [Docker For Windows](https://www.docker.com/products/docker#/windows) to run your Docker containers locally
+
+## Installation and setup
+Download and install the [Visual Studio Tools for Docker](https://visualstudiogallery.msdn.microsoft.com/0f5b2caa-ea00-41c8-b8a2-058c7da0b3e4) from the [Visual Studio Gallery](http://visualstudiogallery.msdn.microsoft.com/).
+
+A required configuration is to setup **[Shared Drives](https://docs.docker.com/docker-for-windows/#/shared-drives)** in Docker for Windows. The setting is required for the volume mapping and debugging support.
+
+Right click the Docker icon in the System Tray, click Settings and select Shared Drives.
+
+![Shared Drives](./media/visual-studio-tools-for-docker/settings-shared-drives-win.png) 
+
+## Create an ASP.NET Web Application and add Docker Support
+
+Using Visual Studio, create a new ASP.NET Core Web Application. When the application is loaded, either select **Add Docker Support** from the **Project Menu** or right click the project from the Solution Explorer and select Add > Docker Support.
+
+Project Menu
+
+![Project Add Docker Support](./media/visual-studio-tools-for-docker/project-add-docker-support.png)
+
+Project Context Menu
+
+![Right Click Add Docker Support](./media/visual-studio-tools-for-docker/right-click-add-docker-support.png)
+
+The following files are added the to the project.
+
+- **Dockerfile**: the Docker file for ASP.NET Core applications is based on the [microsoft/aspnetcore](https://hub.docker.com/r/microsoft/aspnetcore) image. This image includes the ASP.NET Core NuGet packages, which have been pre-jitted improving startup performance. When building .NET Core Console Applications, the Dockerfile FROM will reference the [microsoft/dotnet:1.0.1](https://hub.docker.com/r/microsoft/dotnet) image.
+- **docker-compose.yml**: base Docker Compose file
+- **docker-compose.dev.debug.yml**: Docker Compose file with additional settings for ASPNETCORE_ENVIRONMENT, volume mapping and ---
+title: Visual Studio Tools for Docker
```

Comment:
there's something odd with this topic. You restart the topic several times with metadata sections in the middle of the text.